doc/source/extending-jupyterhub.rst

```diff
@@ -176,3 +176,26 @@ report the current ``Ready/NotReady`` status of all nodes in the cluster.
      size for the workshop. This workflow also helps you avoid scrambling on
      the workshop day to set up the cluster and JupyterHub.
    - **After the workshop:** The cluster can be deleted.
+
+
+Setting up HTTPS
+----------------
+
+Zero to JupyterHub makes setting up HTTPS quite easy.
+
+1. Buy a domain name from a registrar. Pick whichever one you want
+2. Create an `A record` from the domain you want to use, pointing to the
+   external IP provided to the `proxy-public` service.
+3. Wait for the change to propagate. Propagation can take several minutes to
+   several hours. Wait till you can type in the name of the domain you bought,
+   and it shows you the hub.
+4. Add the following to your ``config.yaml``:
+
+   ..code-block:: yaml
+
+     proxy:
+       hosts:
+         - <your-domain-name>
+
+5. Apply the config changes by running ``helm upgrade``.
+6. Wait for about a minute, now your hub is HTTPS enabled!
```

Comment:
Related to the comment at the top, but we should say what this means to a deployer

Reply:
I've added some text here that's vague and buzzworddy, since I don't think we can summarize why you care about HTTPS in sufficient detail without being vague and buzzwordy. I think it's ok for us though.